kong/clustering/rpc/utils.lua

```diff
@@ -12,13 +13,26 @@ local snappy_compress = snappy.compress
 local snappy_uncompress = snappy.uncompress
 
 
+local cap_names = lrucache.new(100)
+
+
 function _M.parse_method_name(method)
```

Comment:
Are all method names predefined? Or these can be generated dynamically?

If so, we can put all defined names into a Lua table to skip the parsing.

Reply:
Currently the method name must be registered to RPC module on-demand, using a static lua table may not be a possible way till now.